src/Sparrow/Json/JsonOperationContext.cs

```diff
@@ -325,6 +325,7 @@ public JsonOperationContext(int initialSize, int longLivedSize, SharedMultipleUs
                 TryDispose(_documentBuilder);
                 TryDispose(_arenaAllocator);
                 TryDispose(_arenaAllocatorForLongLivedValues);
+                TryExecute(_allocateStringValues.Clear);
```

Comment:
shouldn't we also set it to null?

Reply:
IMO, clear is enough here, we don't want to expose ourselves to potential NREs